packages/codemods/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "@compiled/codemods",
   "version": "0.3.2",
-  "description": "Codemods (with JSCodeshift) for Compiled CSS-in-JS.",
+  "description": "A familiar and performant compile time CSS-in-JS library for React.",
```

Comment:
Shouldn't the description be specific to the package?

Reply:
Probably - but this is the outlier ATM. Every other package uses the same description.